src/drtsans/configuration/schema/EQSANS.json

```diff
@@ -144,6 +147,10 @@
           "examples": ["Monitor", "Total charge", "Time"],
           "default": "Total charge"
         },
+        "blockedBeamRunNumber": {
```

Comment:
use validator `onlyOneTrue` instead of `evaluateCondition`. See options `useLogSlice` and `useTimeSlice` in file `common.json` for an example usage.
Meaning of validator `onlyOneTrue` is explained in docstring of function `_validate_only_one_true()`